docs/sources/tempo/traceql/trace-structure.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Trace structure and TraceQL
+menuTitle: Trace structure and TraceQL
+description: Learn about trace structure and TraceQL queries.
+weight: 200
+keywords:
+  - syntax
+  - TraceQL
+---
+
+# Trace structure and TraceQL
+
+The TraceQL language is designed to use the same syntax and semantics as PromQL and LogQL, where possible.
```

Comment:
This doesn't strike me as exactly true since much of the syntax has diverged, though I'm not sure what to say in this respec.

Reply:
how's this? 
```suggestion
Inspired by PromQL and LogQL, TraceQL uses similar the same syntax and semantics .
```

Reply:
Yeah, something like that reads better to me.